mapfishapp/src/main/webapp/app/addons/cadastre/js/main.js

```diff
@@ -543,8 +544,9 @@ GEOR.Addons.Cadastre = Ext.extend(GEOR.Addons.Base, {
             title: OpenLayers.i18n("tab2title"),
             items: fields,
             listeners: {
-                "afterrender": function(form) {
-                    fields[0].focus('', 50);
```

Comment:
This is weird because actually you focus the Tab1 first field when Tab2 is created